docs/replication-manager/replication-manager-for-pxc.md

```diff
@@ -0,0 +1,280 @@
+# Replication manager for Percona XtraDB Cluster
+
+The replication manager script helps manage [multi-source replication](../glossary.md#multi-source-replication) between multiple Percona XtraDB Cluster clusters.
+
+## Preconditions
+
+For example, we deploy multi-source replication links between three data centers: DC1, DC2, and DC3. In each data center, there are three Percona XtraDB Cluster nodes form distinct Galera clusters. The data centers contain the following nodes:
+
+|  DC1  |  DC2  |  DC3  |
+| ----- | ----- | ----- |
+| DC1-1 | DC2-1 | DC3-1 |
+| DC1-2 | DC2-2 | DC3-2 |
+| DC1-3 | DC2-3 | DC3-3 |
+ 
+The topology is the following:
+
+```text
+DC2 <=> DC1 <=> DC3
+```
+
+DC1 is a replica of DC2 and DC3. DC2 and DC3 are replicas of DC1.
+
+The example of the minimal MySQL configuration file for DC1 that is common for all three DC1 nodes:
+
+```text
+[mysqld]
+# General galera reqs
+default_storage_engine=InnoDB
+innodb_autoinc_lock_mode=2
+
+# Replication settings
+binlog_format=ROW
+server-id=1
+log-bin=mysql-bin
+log_slave_updates
+expire_logs_days=7
+gtid_mode = ON
+enforce_gtid_consistency=ON
+master_info_repository = TABLE
+relay_log_info_repository = TABLE
+skip-slave-start
+    
+# Galera configuration
+wsrep_provider=/usr/lib/galera3/libgalera_smm.so
+wsrep_cluster_address=gcomm://10.0.4.160,10.0.4.162,10.0.4.163
+wsrep_slave_threads= 2
+wsrep_log_conflicts
+wsrep_cluster_name=DC1
+pxc_strict_mode=ENFORCING
+wsrep_sst_method=xtrabackup-v2
+wsrep_sst_auth="root:root"
+```
+
+All nodes have the same `server-id` value. The repositories are set to `TABLE` because the multi-source replication syntax is used. Multi-source replication allows a node to become a replica in more than one remote cluster.
```

Comment:
In a multi-source replication topology, the replicas require the `master_info_repository` and `relay_log_info_repository` to be TABLE. These settings cannot be `FILE`.

Reply:
Updated